Library/Homebrew/utils/analytics.sh

```diff
@@ -14,17 +22,15 @@ setup-analytics() {
   if [[ "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsmessage)" != "true" ||
         "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsdisabled)" = "true" ]]
   then
```

Comment:
I think it would be good to just remove it, actually. Err on the side of paranoia 😉